cosmic_ray/util.py

```diff
@@ -51,3 +51,18 @@ def get_line_number(node):
         return node.lineno
     else:
         return '<UNKNOWN>'
+
+def build_mutations(ops, to_ops):
```

Comment:
I think it might be helpful to add an argument `include_identity=False` which, if false, would not return an identity mutation (i.e. where `to_op` and `from_op` are the same). It seems likely that most users of this function will never want to make the identity mutation, so we could make their lives a bit simpler by not producing it.

We could make change in a separate commit, of course.